### ISSUE TITLE
Add a bower.json

### ISSUE THREAD

**Issue by CMCDragonkai** [sdecima/javascript-detect-element-resize#3]
Would make it easier to include.

**Comment by ledzep2** [comment]
+1

**Comment by tiye** [comment]
I did this.. hope it helps..

```
bower resigter detect-element-resize <this repo>
```

**Comment by batista** [comment]
:+1: I'm using [angular-gridster](https://github.com/ManifestWebDesign/angular-gridster) (that uses detect-element-resize as a dependency) on my webapp and it sucks to have to manually insert the reference to your files because bower fails to understand which file to use.

If someone is facing the same issue and getting their grunt builds failing because of this, a quick fix might be to override the main file reference in your `bower.json`:

``` javascript
{
  "name": "yourawesomeappname",
  "version": "0.0.0",
  // ...
  "dependencies": {
    // ...
  },
  "overrides": {
    "javascript-detect-element-resize": {
      "main": "jquery.resize.js"
    }
  },
  "devDependencies": {
    // ...
  }
  // ...
}
```

**Comment by simplesmiler** [comment]
+1

**Comment by sdecima** [comment]
`bower.json` is now included, starting with v0.5.2. Thanks all!
